<?xml version="1.0"?>
<launch>
	<!-- Nodes -->
	<node name="image_view" pkg="image_view" type="image_view" args="image:=/image" output="screen"/>
	<node name="show_image" pkg="publish_media" type="publish_image_demon.py">
	
		<param name="img_path" value="$(find publish_media)/media/test_image.jpg"/>
		<param name="ignore_not_found" value="True"/>

		<param name="lookup/Run" value="$(find publish_media)/media/pic1.jpg"/>
		<param name="lookup/Faster" value="$(find publish_media)/media/pic2.jpg"/>
		<param name="lookup/EvenFaster" value="$(find publish_media)/media/pic3.jpg"/>
		<param name="lookup/Stop" value="$(find publish_media)/media/pic4.jpg"/>

		<remap from="image_trigger" to="info"/>
	</node>
	
	<node name="String_publisher" pkg="rviz_text_publisher" type="string_publisher.py" output="screen"/>

</launch>
